proj/src/main/webapp/chat.js

```diff
@@ -318,18 +318,33 @@ const LIMIT = 20; // how many messages to load at a time
 function initChat() {
     firebase.auth().onAuthStateChanged(async firebaseUser => {
         if(firebaseUser){
-            setupSidebar();
-            clickWithEnterKey();
-            notifications.initNotifications();
-
-            // InitUserChat sets information relevant to logged-in user
-            // Must run before enclosed functions
-            await initUserChat().then(function(){
-                populateSidebar();
-                initRef();
-                populateProfileSidebar(firebaseUser.uid);
-                addBlockedToSettings();
-            });
+            // If the user is verified or using a test server
+            if(firebaseUser.emailVerified || window.location.href.includes("https://8080")){
+                if(firebaseUser.emailVerified)
+                    console.log("You have been verified as a Camaraderie user");
+                else
+                    console.log("You have been verified as a Camaraderie testing developer");
```

Comment:
Can you put curly braces around the parts in the if/else. It probably doesn't need it, but just for my sanity.